modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json

```diff
@@ -419,6 +419,3114 @@
 			"initialReferenceDate": "2022-12-01T00:00:00",
 			"productExternalReferenceCode": "RAYAP-001",
 			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-12-31T00:00:00Z",
+			"goalValue": "20000",
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-11-31T00:00:00Z",
+			"goalValue": "15000",
```

Comment:
This value should be a number. Please change it.

Reply:
Done